mungegithub/github/github.go

```diff
@@ -1611,9 +1611,13 @@ func (obj *MungeObject) ListComments(withListOpts ...WithListOpt) ([]*github.Iss
 // WriteComment will send the `msg` as a comment to the specified PR
 func (obj *MungeObject) WriteComment(msg string) error {
 	config := obj.config
-	prNum := *obj.Issue.Number
+	prNum := obj.Number()
 	config.analytics.CreateComment.Call(config, nil)
-	glog.Infof("Commenting %q in %d", msg, prNum)
+	comment := msg
+	if len(comment) > 512 {
+		comment = comment[:512]
```

Comment:
oh! sorry. I totally missed that this was just the log message.

/lgtm

On Tue, Aug 23, 2016 at 6:46 AM, Eric Paris <notifications@github.com>
wrote:

> In mungegithub/github/github.go
> <https://github.com/kubernetes/contrib/pull/1593#discussion_r75868318>:
>
> >  	config.analytics.CreateComment.Call(config, nil)
> > -	glog.Infof("Commenting %q in %d", msg, prNum)
> > +	comment := msg
> > +	if len(comment) > 512 {
> > +		comment = comment[:512]
>
> I'm not sure exactly what you think would be ok to truncate. This is only
> truncating what goes into the kubectl logs, (obviously not what goes in
> the issue itself)
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/contrib/pull/1593/files/41e54353e4409e02c865cf72d49f9af7cc461bb4#r75868318>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAnglrX1M-WWjIOdbG3ZakOLvnwL7vTCks5qivnRgaJpZM4Jp_G5>
> .
>